test/marketsDistribution.test.ts

```diff
@@ -21,3 +24,67 @@ describe("Markets distribution", () => {
     expect(marketDistribution.snapshotProposal).toBeTruthy();
   });
 });
+
+describe("Age 1 Epoch 2 approximation", () => {
+  const onChainDistribution = require("../distribution/fromDeprecatedScript/proofs-2.json");
+  const newDistribution = require("../distribution/proofs/proofs-2.json");
+
+  it("Should have the same number of users", () =>
+    expect(Object.keys(onChainDistribution.proofs).length).toEqual(Object.keys(newDistribution.proofs).length));
+
+  it("Should contains the same users", () => {
+    const newDistriUsers = Object.keys(newDistribution.proofs);
+    Object.keys(onChainDistribution.proofs).forEach((onChainUser) => expect(newDistriUsers).toContain(onChainUser));
+  });
+  it("Should derive of 7e-18 only", () => {
+    const newDistriUsers = Object.entries(newDistribution.proofs);
+    Object.entries(onChainDistribution.proofs).forEach(([onChainUser, onChainDistrib]) => {
+      const [, compareTo] = newDistriUsers.find(([u]) => u === onChainUser)!;
+      // @ts-ignore unknown type
+      const compareToAmount = BigNumber.from(compareTo.amount);
+      // @ts-ignore unknown type
+      const onChainAmount = BigNumber.from(onChainDistrib.amount);
+      const dust = compareToAmount.gte(onChainAmount)
+        ? compareToAmount.sub(onChainAmount)
+        : onChainAmount.sub(compareToAmount);
+      // the max derivation from script version one and script version two is 7 * 1e-18 $MORPHO
+      expect(dust).toBnLte(7);
+    });
+  });
+
+  describe("Age 1 Epoch 2 approximation", () => {
+    const onChainDistribution = require("../distribution/fromDeprecatedScript/proofs-2.json");
+    const newDistribution = require("../distribution/proofs/proofs-2.json");
+
+    it("Should have the same number of users", () =>
+      expect(Object.keys(onChainDistribution.proofs).length).toEqual(Object.keys(newDistribution.proofs).length));
+
+    it("Should contains the same users", () => {
+      const newDistriUsers = Object.keys(newDistribution.proofs);
+      Object.keys(onChainDistribution.proofs).forEach((onChainUser) => expect(newDistriUsers).toContain(onChainUser));
+    });
+    it("Should derive of 7e-18 only", () => {
```

Comment:
The test is a copy/paste of the test above, could you make a function of its body? (It's better for reviewers.)